src/commands/redis_cmd.cc

```diff
@@ -393,15 +358,18 @@ class CommandGet : public Commander {
 class CommandGetEx : public Commander {
  public:
   Status Parse(const std::vector<std::string> &args) override {
-    white_list_ = {{"persist", false}};
-    auto s = ParseTTL(std::vector<std::string>(args.begin() + 2, args.end()), &white_list_, &ttl_);
-    if (!s.IsOK()) {
-      return s;
-    }
-    if (white_list_["persist"] && args.size() > 3) {
-      return Status(Status::NotOK, errInvalidSyntax);
+    CommandParser parser(args, 2);
+    std::string_view ttl_flag;
+    while (parser.Good()) {
+      if (auto v = GET_OR_RET(ParseTTL(parser, ttl_flag))) {
+        ttl_ = *v;
+      } else if (parser.EatEqICaseFlag("PERSIST", ttl_flag)) {
+        persist_ = true;
+      } else {
+        return parser.InvalidSyntax();
+      }
     }
-    return Commander::Parse(args);
+    return {};
```

Comment:
Personally, I'd like to see more verbose but explicit `Status::OK()` instead of `{}`.

Reply:
done